notebooks/lerobot_act/act_to_openvino.ipynb

```diff
@@ -0,0 +1,598 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Export ACT Model to ONNX
</pre><p>Can we skip this step, and convert it to OV IR directly ?<a href="https://docs.openvino.ai/2025/openvino-workflow/model-preparation/convert-model-pytorch.html" target="_blank">https://docs.openvino.ai/2025/openvino-workflow/model-preparation/convert-model-pytorch.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/3111/discussion/#comment-2488270780'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Added another section to also convert it to OV IR directly.